coffee_cmd/src/main.rs

```diff
@@ -56,7 +56,21 @@ async fn main() -> Result<(), CoffeeError> {
             let remotes = coffee.list().await;
             coffee_term::show_list(remotes)
         }
-        CoffeeCommand::Upgrade => coffee.upgrade(&[""]).await,
+        CoffeeCommand::Upgrade { repo } => {
+            let result = coffee.upgrade(&repo).await;
+            match result {
+                Ok(res) => match res.status {
+                    UpgradeStatus::UpToDate => {
+                        term::info!("Remote repository {} is already up to date!", res.repo)
+                    }
+                    UpgradeStatus::Updated => {
+                        term::success!("Remote repository {} was successfully upgraded!", res.repo)
+                    }
+                },
+                Err(err) => term::error(format!("{err}")),
```

Comment:
```suggestion
                Err(err) => Err(err),
```

We are printing the error already outside the match, right?

Reply:
Yes